### PR TITLE
Fix commit history with upstream

### PR DESCRIPTION
**Pull request by DavidTheProgrammer**
## Description

This commit is necessary only to sync the git histories of the two projects. The initial update appears to have used a rebase thus creating new commits via playback and not actually merging the branches. The initial fix from this branch did however carry over all the file changes and hence the tests from the first time around are still valid. The only file change is the `README.md` file with a removal of the `collectstatic` command.

## Related Issue
#4 

## How to test it locally

Check #5 

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,8 +50,6 @@ Local development is normally done inside docker-compose so that the supporting
 Make docker-compose start the supporting services
 
     docker-compose run --rm web python wait_for_postgres.py
-    docker-compose run --rm web python manage.py collectstatic --no-input
-
```
Comment (kilemensi) [inline README.md:54]: Does it mean this is run "somewhere" else or why don't we need it documented?

**Review by kilemensi — APPROVED** [review]
LGTM (but I don't have all the context).

File: README.md (hunk 1 above)
Comment (satoru260) [reply README.md:54]: This is part of the local run configuration that I'll be re-documenting. As for now, we don't need to actually run this command as we don't have static resources really, so it works fine without it.

**Comment by satoru260** [comment]
> LGTM (but I don't have all the context).

Essentially we're just updating the commit history so that we have all commits from upstream. The change in the `README.md`, though valid, is just there because I wasn't sure if Github could create a PR with no file changes.

**Comment by kilemensi** [comment]
> Essentially we're just updating the commit history so that we have all commits from upstream. The change in the `README.md`, though valid, is just there because I wasn't sure if Github could create a PR with no file changes.
>

Cool @DavidTheProgrammer , are we planning on re-adding it in another PR or does the app run fine without the `collectstatic` command (in `docker compose`, I know Dokku has `app.json` to configure it).

**Comment by kilemensi** [comment]
... saw the "This is part of the local run configuration that I'll be re-documenting." comment so I'm assuming it's a separate PR.

**Comment by satoru260** [comment]
> are we planning on re-adding it in another PR or does the app run fine without the collectstatic command

The app runs fine without the `collectstatic` command at present.

> ... saw the "This is part of the local run configuration that I'll be re-documenting." comment so I'm assuming it's a separate PR.

No, it won't be re-added to the document in the other PR. But I will improve the README docs in that separate PR to reflect the current state of running it locally.

**Comment by kilemensi** [comment]
> The app runs fine without the collectstatic command at present.
>

Unlikely @DavidTheProgrammer , otherwise, there wouldn't have been a need of this pre-deploy [script](https://github.com/CodeForAfrica/wazimap-ng/blob/staging/scripts/django/predeploy.sh) (that Dokku calls [here](https://github.com/CodeForAfrica/wazimap-ng/blob/df67be29d82e0721c158ee8ade4f111820ba6a83/app.json#L4)).

What's the equivalent in `docker compose` world?

**Comment by satoru260** [comment]
> > The app runs fine without the collectstatic command at present.
> 
> Unlikely @DavidTheProgrammer , otherwise, there wouldn't have been a need of this pre-deploy [script](https://github.com/CodeForAfrica/wazimap-ng/blob/staging/scripts/django/predeploy.sh) (that Dokku calls [here](https://github.com/CodeForAfrica/wazimap-ng/blob/df67be29d82e0721c158ee8ade4f111820ba6a83/app.json#L4)).
> 
> What's the equivalent in `docker compose` world?

Maybe my memory is failing me but I don't think I ever ran a `collectstatic` command and the app worked fine. I'll have to doublecheck these assumptions when rewriting the docs. In compose we can achieve something similar by providing multiple commands to the service:

```yaml
...
    command: >
      bash -c "
      python manage.py collectstatic --noinput 
      && python manage.py runserver_plus 0.0.0.0:8000 --keep-meta-shutdown
      "
```
This works as expected. Other options include a custom startup script to execute as the command.
